scalding-base/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -322,20 +327,6 @@ object Execution {
   def withConfig[T](ex: Execution[T])(c: Config => Config): Execution[T] =
     TransformedConfig(ex, c)
 
-  /**
-   * Distributes the file onto each map/reduce node, so you can use it for Scalding source creation and
-   * TypedPipe, KeyedList, etc. transformations. Using the [[com.twitter.scalding.filecache.CachedFile]]
-   * outside of Execution will probably not work.
-   *
-   * For multiple files you must nested your execution, see docs of
-   * [[com.twitter.scalding.filecache.DistributedCacheFile]]
-   */
-  def withCachedFile[T](path: String)(fn: CachedFile => Execution[T]): Execution[T] =
```

Comment:
this will almost certainly need to be added back, and I think it can be.

Reply:
This I know is used at Twitter, although maybe not much. For some things like credentials I made it part of the external tooling to run the Hadoop command with distributed files, so it might be possible to remove it from Scalding if all uses followed that pattern.